src/Components/Navigation/NavigationBar.css

```
@@ -0,0 +1,73 @@
+ul {
+    list-style: none;
+    color: rgba(255, 255, 255, 100);
```

opacity (alpha) is in dit geval overbodig (4e value)
```suggestion
    color: rgb(255, 255, 255);
```
of nog leesbaarder:
```suggestion
    color: #fff;
```